### PR TITLE
Adding sample yaml files for mounting secrets as file

### PR DESCRIPTION
**Pull request by akashsinghal**
## Purpose

* Add documentation for new feature that allows secrets to be mounted as files

## Does this introduce a breaking change?
```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

**Comment by shahzzzam** [comment]
@sajayantony @northtyphoon @SteveLasker

File: mounts-multistep.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# This template demonstrates mounting custom volumes into a container for multi-step task
+volumes:
+  - name: secret1
+    secret:
+      mysecret1: {{"-----SECRET VALUE 1------" | b64enc}}
```
Comment (northtyphoon) [inline mounts-multistep.yaml:5]: can we use something like  {{ .Values.secret1 | b64enc }}? We don't want to encourage the customers to bake the secret string as plain text in the template =)

Comment (akashsinghal) [reply mounts-multistep.yaml:5]: would I need to provide a corresponding Values file or is it ok to assume they would know how to use that?

Comment (shahzzzam) [reply mounts-multistep.yaml:5]: no need to provide values file

**Pull request merged**